rebar.config

```diff
@@ -1,5 +1,5 @@
 %-*-Erlang-*-
-{erl_opts, [debug_info]}.
+{erl_opts, [debug_info]}. % bin_opt_info
```

Comment:
Perhaps a comment like this:
''passing the option through the environment (ERL_COMPILER_OPTIONS=bin_opt_info) is in most cases the most practical approach.''
Or a link to http://erlang.org/doc/efficiency_guide/binaryhandling.html#option-bin_opt_info